components/layout_2020/flexbox/layout.rs

```diff
@@ -707,15 +713,15 @@ impl FlexContainer {
         let mut cross_start_position_cursor = match resolved_align_content {
```

Comment:
I guess this could use `resolve_reversable_flex_alignment()` and remove the cases for `AlignFlags::FLEX_START`  and `AlignFlags::FLEX_END`.